app/src/main/java/com/example/diceroller/RolledDicesListFragment.kt

```diff
@@ -1,38 +1,61 @@
 package com.example.diceroller
 
-import androidx.lifecycle.ViewModelProvider
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
 import com.example.diceroller.databinding.RolledDicesListFragmentBinding
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.flow.filter
+import kotlinx.coroutines.flow.take
+import kotlinx.coroutines.launch
 
 class RolledDicesListFragment : Fragment() {
```

Comment:
⛏️  Nitpick: Because Dice is the plural form, you don't need the "s"

```suggestion
class RolledDiceListFragment : Fragment() {
```